---
authors: Freek Pols
updated: December 3, 2025
---
# Cappucino

## Introductie
Heb je al eens bedacht dat het snel verwarmen van melk voor een cappucino eigenlijk heel snel gaat zonder dat de melk verdunt wordt?
De melk wordt ook niet echt plaatselijk verhit door een warmte element.
In plaats daarvan wordt er stoom door de melk geblazen.
De stoom condenseert in de melk en geeft daarbij zijn latente warmte af.
Hierdoor warmt de melk snel op zonder dat er (significant veel) water aan toegevoegd wordt.

```{warning}
Stoom is heel heet en kan voor vervelende brandplekken zorgen.
Wees dus voorzichtig bij het uitvoeren van dit practicum.

Daarnaast, nadat de kookplaat uitgaat is er een onderdruk in de kolf.
Hang dan ook de tube uit het water wanneer je de kookplaat uitzet, anders kan de kolf imploderen.
```

In dit practicum gaan we de verdampingswarmte van water bepalen door middel van een zelfgebouwde cappucino machine.
Het principe is als volgt: We gebruiken de opstelling weergegeven in {numref}`Figuur {number} <fig_cappucino>` waarbij we water verwarmen in een kolf met behulp van een kookplaat.
Door de warmte gaat het water koken en ontstaat er stoom.
De stoom stroomt via een tube naar een afgesloten maatbeker met water.
De stoom condenseert in het maatbeker en geeft daarbij zijn latente warmte af.
Door de temperatuurstijging van het water te meten, evenals de hoeveelheid gram water dat verdampt is, kunnen we de verdampingswarmte van water bepalen.

```{figure} figures/verdampingswarmte_setup.png
:width: 70%
:label: fig_cappucino

Een schematische weergave van de cappucino opstelling.
```

## Theorie

De latente verdampinswarmte van water bedraagt 2257 kJ/kg, dit is veel meer dan de specifieke warmtecapaciteit van water (4.18 kJ/kgK).
Wanneer we de waterdamp (g) door koud water leiden, gaan we er van uit dat het waterdamp condenseert en daarbij zijn latente warmte afgeeft aan het koude water.
Door te bepalen hoeveel gram water verdampt is en hoeveel de temperatuur van het koude water stijgt, kunnen we de latente warmte van verdamping bepalen:

$$ Q_{condensatie} = m_{damp} L = m_{water} c \Delta T $$

met $m_{damp} = \Delta m_{kolf}$.

## Methoden en materialen

### Materialen
- Warmteplaat
- Kolf met stop en tube
- Maatcilinder
- Thermometer
- Weegschaal
- Water


### Procedure

- Vul de kolf met ongeveer 100 mL water: bepaal precies de massa water ($m_{w_1}$).  
- Vul de maatcilinder met ongeveer 100 mL water: bepaal precies de massa water ($m_{bad}$).
- Bepaal de temperatuur van dit waterbad ($T_{bad_1}$).
- Zet de kolf op de warmteplaat en zet de warmteplaat aan - maximale stand 3, zorg ervoor dat de tube goed in het waterbad hangt.
- Wacht tot de temperatuur van het waterbad met ongeveer 20 K is gestegen.
- Stop de meting / zet de warmteplaat uit. Noteer meteen de temperatuur($T_{bad_2}$) en haal de tube uit het water! 

```{warning}
Pas op stoom is heet!
```

- Bepaal precies de massa water in de kolf ($m_{w_2}$).
- Verwerk je resultaten hieronder om de latente warmte van verdamping van water te bepalen.

## Resultaten

In [7]:
#Massa van water in kolf
m_w_kolf_leeg = 110.6  # g
m_w_kolf_vol = 210.3  # g

# massa water in maatcilinder
m_maatcilinder_leeg = 136.362  # g
m_maatcilinder_vol = 235.185  # g

#soortelijke warmte van water
c_w = 4180  # J/(kg·K)

# berekeningen massa water in kolf en maatcilinder aan het begin van de proef
m_w_1_kolf =( m_w_kolf_vol- m_w_kolf_leeg) /1000 # kg
print(f"Massa water in kolf is {m_w_1_kolf:.2f} kg")
m_w_bad= (m_maatcilinder_vol - m_maatcilinder_leeg) / 1000   # kg
print(f"Massa water in maatcilinder is {m_w_bad:.2f} kg")
T_bad_1 =18.2 + 273.15 # K

# massa water in kolf aan het einde van de proef
m_w_2_kolf = (203.2 - 110.6) /1000   # kg
print(f"Massa water in kolf na verwarmen is {m_w_2_kolf:.2f} kg")
# eindtemperatuur bad
T_bad_2 = 41.8  + 273.15  # K
# massa waterdamp gevormd in kolf
m_damp = m_w_1_kolf - m_w_2_kolf  # kg
print(f"Massa waterdamp gevormd is {m_damp:.4f} kg")

# delta T bad en delta T condenswater
# condenswater koelt namelijk gedurende de weg door de buis af van 100 °C naar T_bad_2
T_stoom_C = 100 + 273.15  # K
dT_bad = T_bad_2 - T_bad_1                   # K of °C
dT_condenswater = T_stoom_C - T_bad_2         # K (afkoeling van 100 naar eindtemp)

# Energiebalans
# De stoom geeft energie af via condensatie (m*L) EN door af te koelen naar T_bad_2.
# Q_opgenomen_water = Q_vrijgekomen_condensatie + Q_vrijgekomen_afkoeling
Q_totaal_opgenomen = m_w_bad * c_w * dT_bad
Q_afkoeling_condens = m_damp * c_w * dT_condenswater

# Latente warmte L berekenen: L = (Q_totaal - Q_afkoeling) / m_damp
L_berekend = (Q_totaal_opgenomen - Q_afkoeling_condens) / m_damp
print(f"Latente warmte van verdamping is {L_berekend/1000:.2f} kJ/kg")

Massa water in kolf is 0.10 kg
Massa water in maatcilinder is 0.10 kg
Massa water in kolf na verwarmen is 0.09 kg
Massa waterdamp gevormd is 0.0071 kg
Latente warmte van verdamping is 1129.78 kJ/kg


## Discussie en conclusie

De berekende waarde ($L \approx 1130$ kJ/kg) wijkt aanzienlijk af van de literatuurwaarde (2257 kJ/kg). De hoofdoorzaak is warmteverlies aan de omgeving en de maatbeker zelf. Hoewel de kolf en de maatcilinder van glas zijn – een materiaal dat relatief slecht geleidt en daardoor een beperkte isolerende werking heeft – is het systeem niet volledig geïsoleerd. Een significant deel van de latente warmte is verloren gegaan aan het opwarmen van de lucht en de wand van de maatcilinder zelf. In de berekening is alleen de massa van het water meegenomen. De energie die nodig was om de maatcilinder (het glas) in temperatuur te laten stijgen, is niet verwerkt in de energiebalans. Hierdoor lijkt de vrijgekomen energie van de stoom, en dus de latente warmte, lager dan deze in werkelijkheid was. Daarnaast is de massa van de stoom is erg klein (ca. 7 gram). Een kleine afwijking op de weegschaal heeft hierdoor een enorme invloed op het eindresultaat. Verder legt De stoom een weg af door de slang voordat deze het waterbad bereikt. Onderweg treedt al enige condensatie op door afkoeling, waardoor de stoom bij aankomst in het bad mogelijk al deels vloeibaar is en dus minder latente warmte kan overdragen. In een vervolgonderzoek zou een calorimeter (een vacuüm geïsoleerd vat) gebruikt moeten worden om het warmteverlies naar de omgeving te minimaliseren.

